src/coreclr/vm/cachelinealloc.h

```diff
@@ -65,7 +65,8 @@ class CacheLine
         CONTRACTL_END;
 
         // initialize cacheline
-        memset(&m_Link,0,32);
+        m_Link = {};
```

Comment:
The caller of this method wants to clear 32 bytes worth of data. `m_Link` is not used.

Would it be better to clear the memory using `memset` on the pointer cast to `void*` directly in the caller; and delete this method?

Reply:
Sure. I tweaked the code a bit since this required some more inspection to really get what it was being used for. I assume this data structure had far more uses in the past.